### DISCUSSION TITLE
Reward System

### DISCUSSION THREAD

**Discussion by LukasKraiger** [KL-Psychological-Methodology/ESMira#24] [General]
Hi,

i am using the ESMira App for a Study at the Moment and in the Reward System the auto Code insert doesnt work at least on the Iphone on other Platforms I could test it. Here is the Text I use:

This is my  [[CODE]]  please send me the link to enter my bank details.

Did I do something wrong?

Best
Lukas Kraiger

**Comment by SelinaDev** [comment]
Hi Lukas,

thank you for your question. I just tested it, this seems to be a bug on iOS. I've opened an issue (#25) and will look into it. If all works out I'll just include the fix in the next version of the iOS app.

Best,
Selina

**Comment by LukasKraiger** [comment]
Hi,

thank you very much. Would this be an update for the ios App or the webapp?

Nevermind I just saw it in your commit. Thank you.

Best
Lukas

**Reply by SelinaDev** [reply]
The update will be for the iOS app. The replacement simply was not happening in the code, but was added now. It was just one line that we forgot to add, so it was a quck fix.
As soon as it's available in the app store and users have updated to that version it should work. There's nothing you need to do or change. Unless my plans change somehow I'll probably upload the update next week.

**Reply by LukasKraiger** [reply]
Thank you so much :)
